boards/arm/lpcxpresso55s69/lpcxpresso55s69_ns.yaml

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2019, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+identifier: lpcxpresso55s69_cpu0
```

Comment:
As above, should this be `lpcxpresso55s69_cpu1` if the NS app is running on the second core?

Reply:
@microbuilder lpcxpresso55s69_ns refers to CPU0 built with NS address ranges.
CPU1 is NS only, guess I should document this.